### PR TITLE
PMM-6096 MongoDB connection checking

### PR DESCRIPTION
**Pull request by oter**
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1342

### CODE REVIEW

**Comment by AlekSi** [comment]
@oter Please click that CLA button/badge to sign CLA

File: connectionchecker/connection_checker_test.go [hunk 1]
```diff
@@ -59,12 +59,27 @@ func TestConnectionChecker(t *testing.T) {
 		},
 
 		{
+			name: "MongoDB with no auth",
+			req: &agentpb.CheckConnectionRequest{
+				Dsn:     "mongodb://127.0.0.1:27018/admin?connectTimeoutMS=1000",
```
Comment (AlekSi) [inline connectionchecker/connection_checker_test.go:64]: Let's add another test: with credentials for DB that does not require them

File: docker-compose.yml [hunk 2]
```diff
@@ -57,6 +57,13 @@ services:
       - MONGO_INITDB_ROOT_USERNAME=root
       - MONGO_INITDB_ROOT_PASSWORD=root-password
 
+  mongonoauth:
+    image: ${MONGO_IMAGE:-percona/percona-server-mongodb:4.2}
+    container_name: pmm-agent_mongo_noauth
```
Comment (AlekSi) [inline docker-compose.yml:62]:
```suggestion
    container_name: pmm-agent_mongonoauth
```

Or `mongo_noauth` two lines above. For consistency.

**Review by BupycHuk — APPROVED** [review]

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Tests are failing

**Comment by AlekSi** [comment]
And I think it should not be a draft :)

File: connectionchecker/connection_checker_test.go (hunk 1 above)
Comment (oter) [reply connectionchecker/connection_checker_test.go:64]: added

**Review by BupycHuk — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Is there a feature build?

**Comment by oter** [comment]
@AlekSi the link to FB was added to the PR description.

**Pull request merged**